data_steward/utils/participant_summary_requests.py

```diff
@@ -104,4 +124,57 @@ def get_deactivated_participants(project_id, columns):
     df = pandas.DataFrame(deactivated_participants,
                           columns=deactivated_participants_cols)
 
-    return df
+    # Converts column `suspensionTime` from string to timestamp
+    if 'suspensionTime' in deactivated_participants_cols:
+        df['suspensionTime'] = pandas.to_datetime(df['suspensionTime'])
+
+    # Transforms participantId to an integer string
+    # Rename participantId column to person_id
+    df['participantId'] = df['participantId'].apply(participant_id_to_int)
+    df = df.rename(columns={'participantId': 'person_id'})
```

Comment:
If we are not passing column names and we use underscores as delimiters instead of camel casing, this should be updated to:

```suggestion
    bq_columns = {'participantId': 'person_id', 'suspensionTime': 'suspension_time', 'suspensionStatus': 'suspension_status'}
    df = df.rename(columns=bq_columns)
```

If we do still pass the column names, and want to use camel casing, we can use a regular expression to create a list of strings that are formatted correctly.  Then we can zip the contents together to create a dictionary.  What do you think about this?

Reply:
yes, this would be best, I wanted columns to be passed in because they could possibly change in the future.